Misc/NEWS.d/next/Library/2019-02-15-17-11-13.bpo-35928.zmzAK9.rst

```diff
@@ -0,0 +1 @@
+TextIOWrapper now correctly handles the decoding buffer in read-write mode, using sockets aswell.
```

Comment:
What "using sockets as well" means?

Reply:
In Issue 35928 the initial symptom was about sockets using socket.makefile truncating the output to a client, as seen in the first test case.

Reply:
I removed the note to sockets as this is a side effect.